laterpay/application/Controller/Install.php

```diff
@@ -519,4 +520,57 @@ public function update_default_pricing_behaviour() {
             delete_option( 'laterpay_only_time_pass_purchases_allowed' );
         }
     }
+
+    /**
+     * Init ga options.
+     * Add options needed for sending Google Analytics Events.
+     *
+     * @since 2.2.0
+     *
+     * @return void
+     */
+    public function init_ga_options() {
+        $current_version = get_option( 'laterpay_plugin_version' );
+        // @todo: change version compared to `2.1.0`
+        // @todo: Only Add / Update option only if current one doesn't exist.
+        if ( version_compare( $current_version, '2.0.0', '<' ) ) {
+            return;
+        }
+
+        $lp_personal_data = [];
+        $lp_tracking_data = [];
+
+        $lp_personal_data['laterpay_ga_personal_enabled_status'] = 0;
+        $lp_personal_data['laterpay_ga_personal_ua_id']          = '';
+
+        if ( is_multisite() ) {
+            $monster_insights_network_profile = get_site_option( 'monsterinsights_network_profile', array() );
+            $monster_ua_id                    = $monster_insights_network_profile['ua'];
+        } else {
+            $monster_insights_profile = get_option( 'monsterinsights_site_profile', array() );
+            $monster_ua_id            = $monster_insights_profile['viewname'];
+        }
+
+        if ( ! empty( $monster_ua_id ) ) {
+            $lp_personal_data['auto_detected']              = 1;
+            $lp_personal_data['laterpay_ga_personal_ua_id'] = $monster_ua_id;
+        }
+
+        // Add Personal Google Analytics Data.
+        update_option( 'laterpay_user_tracking_data', $lp_personal_data );
+
+        $lp_is_plugin_live = (bool) get_option( 'laterpay_plugin_is_in_live_mode' );
```

Comment:
If this is used multiple times then we can create a generic function for this and can then use it directly in if condition.